test/tests/pbs_Rrecord_resources_used.py

```diff
@@ -284,25 +312,29 @@ def test_t4(self):
         self.server.rerunjob(jobid=jid2)
         self.server.rerunjob(jobid=jid3s1)
 
-        self.server.expect(JOB, {ATTR_state: 'R'}, jid1, offset=5)
+        self.server.expect(JOB, {ATTR_state: 'R'}, jid1)
         self.server.expect(JOB, {ATTR_state: 'R'}, jid2)
         self.server.expect(JOB, {ATTR_state: 'R'}, jid3s1)
 
         """
         Confirm that the 'R' record is generated and the run_count is 2.
         """
         m = self.server.accounting_match(
-        msg='.*R;' + jid1 + '.*resources_used.*run_count=2', id=jid1, regexp=True)
+            msg='.*R;' + jid1 + '.*resources_used.*run_count=2', id=jid1,
+            regexp=True)
         self.assertNotEqual(m, None)
         m = self.server.accounting_match(
-        msg='.*R;' + jid2 + '.*resources_used.*run_count=2', id=jid2, regexp=True)
+            msg='.*R;' + jid2 + '.*resources_used.*run_count=2', id=jid2,
+            regexp=True)
         self.assertNotEqual(m, None)
-
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'},
+                            expect=True)
```

Comment:
scheduling is set to False in the common(). revert_to_defaults() does not set the scheduling to True once the test case is over. Confirmed it by running the test case by commenting this line, In the subsequent test case, the jobs are not getting scheduled to run.
However, I do feel we should have this in the setUp(). So, I will put it there.